--- /tmp/texlive/texmf-dist/tex/latex/newpax/newpax.lua
+++ /tmp/texlive/texmf-dist/tex/latex/newpax/newpax.lua
@@ -206,7 +206,7 @@
 -- XXXXXX encode/escape the file name?
 local function outputENTRY_file (file, pdfedoc)
   local bytes       = GETSIZE(pdfedoc)
-  local date        = GETINFO(pdfedoc).CreationDate
+  local date        = "D:22222222222222"
   -- file
   local a = strENTRY_BEG
   a = a .. strCMD_BEG .. constCMD_FILE .. strCMD_END
